challenge-02/README.md

```diff
@@ -35,19 +40,28 @@ Crie uma função com as seguintes características:
     Preencha todos os valores corretamente!
 3. O retorno da função deve ser a multiplicação dos 3 argumentos, somando `2` ao resultado da multiplicação.
 */
-?
+function newFunction(x, y, z){
+    if(x == null && y == null && z == null){
```

Comment:
-1
Nesse caso, a sua verificação precisa que todos os valores não sejam passados para retornar a mensagem que os valores devem ser preenchidos. Na verdade, se qualquer um deles não for passado, você já deve mostrar a mensagem :)